### PR TITLE
Damienbod/first web api todo api update to csproj

### PR DESCRIPTION
**Pull request by damienbod**

### CODE REVIEW

**Comment by damienbod** [comment]
@Rick-Anderson @danroth27

File: aspnetcore/tutorials/first-web-api/sample/src/TodoApi/Startup.cs [hunk 1]
```diff
@@ -26,7 +26,11 @@ public Startup(IHostingEnvironment env)
         public void ConfigureServices(IServiceCollection services)
         {
             // Add framework services.
-            services.AddMvc();
+            services.AddMvcCore()
+                .AddAuthorization()
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/first-web-api/sample/src/TodoApi/Startup.cs]: Are all these services (that the sample is not using) recommended for minimal Web API apps?

cc @danroth27

Comment (danroth27) [inline aspnetcore/tutorials/first-web-api/sample/src/TodoApi/Startup.cs]: For our intro to building Web APIs tutorial I think we should stick with `AddMvc` as it creates the least amount of friction.

Comment (damienbod) [inline aspnetcore/tutorials/first-web-api/sample/src/TodoApi/Startup.cs]: ok, reverted

**Comment by Rick-Anderson** [comment]
@damienbod  thanks for upgrading. BTW, I'd really like to update this to use the EF in-memory DB so you get integers (1, 2, 3) as the PK. Makes it much easier to test/experiment when you don't need to copy/paste a GUID.

**Comment by damienbod** [comment]
OK, Should I create a repository with the EF-Core 
ITodoRespository, TodoRespository or use EF-Core directly in the Contontroller?
Greetings Damien

**Comment by Rick-Anderson** [comment]
>Should I create a repository with the EF-Core
ITodoRespository, TodoRespository or use EF-Core directly in the Controller?

 @MikeWasson is the original author - I'm guessing he would prefer you use the repository pattern and where the interface is implemented replace GUID generations with the in-memory EF Core.
 I don't think it adds much complexity.

**Comment by damienbod** [comment]
https://github.com/aspnet/Docs/pull/2760

**Pull request merged**